### DISCUSSION TITLE
Playlist by all users

### DISCUSSION THREAD

**Discussion by pratyushgta** [jagrosh/MusicBot#936] [Questions]
I'm not sure if this has been already answered but I didn't find this anywhere. 

Just wanted to know, why can playlists be only created by owners and not other users as well?

**Comment by qomiter** [comment]
I had to set DJ role to everyone

On Wed, Sep 29, 2021 at 11:55 PM Pratyush Kumar ***@***.***>
wrote:

> I'm not sure if this has been already answered but I didn't find this
> anywhere.
>
> Just wanted to know, why can playlists be only created by owners and not
> other users as well?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jagrosh/MusicBot/discussions/936>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF4UMQ6UDLR4T565RYTQFJLUEPNTBANCNFSM5FBK2GUA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 


-Dan
